### DISCUSSION TITLE
Unknown flag when using command from the docs

### DISCUSSION THREAD

**Discussion by ChaseCares** [authelia/authelia#4228] [Q&A] [answered]
Hello, when I run the command `authelia crypto hash generate pbkdf2 --variant sha512 --random --random.length 72` from the [specific-information](https://www.authelia.com/integration/openid-connect/specific-information/#generating-client-secrets) docs I receive `Error: unknown flag: --variant`. I'm running the command in the docker cli in the authelia container.

Any guidance would be much appreciated, thank you.

authelia version v4.36.9

Full output
```
/app # authelia crypto hash generate pbkdf2 --variant sha512 --random --random.length 72
Error: unknown flag: --variant
Usage:
  authelia crypto [command]

Examples:
authelia crypto --help

Available Commands:
  certificate Perform certificate cryptographic operations
  pair        Perform key pair cryptographic operations

Flags:
  -h, --help   help for crypto

Use "authelia crypto [command] --help" for more information about a command.

/app #
```

**Comment by james-d-elliott** [comment, marked as answer]
You'll need to use `hash-password` for now, the docs are currently at 4.37.0 level but that has not quite been released (today or tomorrow). In the future we plan to configure versioned docs.

**Reply by ChaseCares** [reply]
Oh I'm just a bit early, thanks for the response!
